### PR TITLE
feat(vscode): Add Integration account source and export custom API actions to export advance options

### PR DESCRIPTION
**Pull request by ccastrotrejo**
### Main Code Changes

* Added two new options to the `AdvancedOptions` component: "Make Integration Account the default source for Transformations and Schema validation" and "Export Custom Connector Actions as API Management Actions". These options are also added to the dropdown in the component.
* Added the keys for the new advanced options to the `AdvancedOptionsTypes` object.
* Modified the `getAdvanceOptionsSelection` function to filter out certain options when the `generateInfrastructureTemplates` option is selected.
* Updated the `WorkflowsSelection` component for handling the success of the `loadWorkflows` query to actually use the catching of the useQuery and avoid multiple api calls
* Removed fixed widths from the input and dropdown elements 

#### Screenshots

https://github.com/Azure/LogicAppsUX/assets/102700317/1595a6d2-140c-4c39-b4f2-e9c0a356a4f6

### CODE REVIEW

File: apps/vs-code-react/src/app/export/workflowsSelection/advancedOptions.tsx [hunk 1]
```diff
@@ -42,6 +42,14 @@ export const AdvancedOptions: React.FC = () => {
       defaultMessage: 'Generate infrastructure templates',
       description: 'Generate infrastructure templates',
     }),
+    INTEGRATION_ACCOUNT_SOURCE: intl.formatMessage({
```
Comment (ccastrotrejo) [inline apps/vs-code-react/src/app/export/workflowsSelection/advancedOptions.tsx:45]: @ecfan Could you please take a look at the wording for these strings? 
++ @wsilveiranz to be aware of it

File: apps/vs-code-react/src/app/export/workflowsSelection/advancedOptions.tsx [hunk 2]
```diff
@@ -42,6 +42,14 @@ export const AdvancedOptions: React.FC = () => {
...
+      defaultMessage: 'Make Integration Account the default source for Transformations and Schema validation',
```
Comment (ecfan) [inline apps/vs-code-react/src/app/export/workflowsSelection/advancedOptions.tsx:46]: Consistent capitalization to use sentence case: https://learn.microsoft.com/en-us/style-guide/capitalization

```suggestion
      defaultMessage: 'Default to integration account as source for transformations and schema validation',
```

File: apps/vs-code-react/src/app/export/workflowsSelection/advancedOptions.tsx [hunk 3]
```diff
@@ -42,6 +42,14 @@ export const AdvancedOptions: React.FC = () => {
...
+      description: 'Make Integration Account the default source for Transformations and Schema validation',
```
Comment (ecfan) [inline apps/vs-code-react/src/app/export/workflowsSelection/advancedOptions.tsx:47]:
```suggestion
      description: 'Default to integration account as source for transformations and schema validation',
```

File: apps/vs-code-react/src/app/export/workflowsSelection/advancedOptions.tsx [hunk 4]
```diff
@@ -42,6 +42,14 @@ export const AdvancedOptions: React.FC = () => {
...
+    }),
+    EXPORT_CUSTOM_API_ACTIONS_TO_API_MANAGEMENT_ACTIONS: intl.formatMessage({
+      defaultMessage: 'Export Custom Connector Actions as API Management Actions',
```
Comment (ecfan) [inline apps/vs-code-react/src/app/export/workflowsSelection/advancedOptions.tsx:50]:
```suggestion
      defaultMessage: 'Export custom connector actions as API Management actions',
```

File: apps/vs-code-react/src/app/export/workflowsSelection/advancedOptions.tsx [hunk 5]
```diff
@@ -42,6 +42,14 @@ export const AdvancedOptions: React.FC = () => {
...
+      description: 'Export Custom Connector Actions as API Management Actions',
```
Comment (ecfan) [inline apps/vs-code-react/src/app/export/workflowsSelection/advancedOptions.tsx:51]:
```suggestion
      description: 'Export custom connector actions as API Management actions',
```

**Review by ecfan — CHANGES_REQUESTED** [review]
Feedback added, thanks!

**Review by ecfan — APPROVED** [review]

**Review by hartra344 — APPROVED** [review]

**Pull request merged**